troubadix/standalone_plugins/file_extensions.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+import sys
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import List
+
+exclusions: List[str] = [
+    "/common/bad_rsa_ssh_host_keys.txt",
+    "/common/bad_dsa_ssh_host_keys.txt",
+    "/22.04/bad_rsa_ssh_host_keys.txt",
+    "/22.04/bad_dsa_ssh_host_keys.txt",
+    "/21.04/bad_rsa_ssh_host_keys.txt",
+    "/21.04/bad_dsa_ssh_host_keys.txt",
```

Comment:
```suggestion
```

For now i would leave them out to avoid wrong copy'n'paste mistakes, the list is flexible enough to add them if really required in the future :+1:

Reply:
Implemented in 65427b17369410a73762fcf4eeacb8900393b13a